src/Microsoft.Identity.Web.TokenAcquisition/MsalMtlsHttpClientFactory.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Identity.Client;
+
+namespace Microsoft.Identity.Web
+{
+    internal sealed class MsalMtlsHttpClientFactory : IMsalMtlsHttpClientFactory
```

Comment:
I don't understand the purpose of this class. It derives from IMsalMtlsHttpClientFactory and encapsulates an IMsalMtlsHttpClientFactory . 

I think it is ok if it dervies IMsalMtlsHttpClientFactory but it should encapsulate [System.Net.Http](https://learn.microsoft.com/en-us/dotnet/api/system.net.http?view=net-9.0-pp)IHttpClientFactory